From c3a9a2a0ee1b349081db1c6f8d17771165557c9b Mon Sep 17 00:00:00 2001
From: "Christopher N. Hesse" <raymanfx@gmail.com>
Date: Fri, 21 Sep 2018 23:00:26 +0200
Subject: [PATCH 3/3] common: Allow custom vendor to override properties

Test case: override ro.adb.secure in vendor/extra/product.mk
Before this patch, the override would not affect the resulting
property.

Change-Id: I0deb8a6865030b09ed6ba0ede1658319ca033e67
---
 config/common.mk | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index b8c94db6..c8fe840c 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -1,3 +1,6 @@
+# Allow vendor/extra to override any property by setting it first
+$(call inherit-product-if-exists, vendor/extra/product.mk)
+
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
 
 ifeq ($(PRODUCT_GMS_CLIENTID_BASE),)
@@ -201,8 +204,6 @@ endif
 
 include vendor/bootleggers/config/btlg_main.mk
 
-$(call inherit-product-if-exists, vendor/extra/product.mk)
-
 # RRO Overlays
 PRODUCT_PACKAGES += \
     NavigationBarModeGesturalOverlayFS
-- 
2.30.2

